node/node.go

```diff
@@ -344,8 +355,72 @@ var (
 	errNotRightKeySize = errors.New("key received over wire is wrong size")
 	errNoSenderPubKey  = errors.New("no sender public BLS key in message")
 	errWrongShardID    = errors.New("wrong shard id")
+	errInvalidNodeMsg  = errors.New("invalid node message")
+	errIgnoreBeaconMsg = errors.New("ignore beacon sync block")
 )
 
+// validateNodeMessage validate node message
+func (node *Node) validateNodeMessage(ctx context.Context, payload []byte) (
+	[]byte, proto_node.MessageType, error) {
+	atomic.AddUint32(&node.NumTotalNodeMsg, 1)
```

Comment:
Is it true that these metrics is only increasing? Is there any possibility that integer overflow might happen?

Reply:
The metrics will be reset every 1 minute, so no worries.